docs/dev_utilities.rst

```diff
@@ -0,0 +1,31 @@
+=========
+Utilities
+=========
+
+This document covers the utilities that are availble to developers of libCellML.
+The utilities are not built or made available by default.
+The utilities can be enabled through the configuration of the library.
+In order to enable the utilities set the configuration variable LIBCELLML_ENABLE_UTILITES to true.
```

Comment:
"... utilities, set..."
Replace "LIBCELLML_ENABLE_UTILITES" with "`LIBCELLML_ENABLE_UTILITES`" and "true" with "`TRUE`".